website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -81,7 +81,7 @@ spec:
         volumeType: gp3
         iops: 10000
         encrypted: true
-        kmsKeyID: "arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab"
+        kmsKeyID: "1234abcd-12ab-34cd-56ef-1234567890ab"
```

Comment:
Nice catch! Can you make this change in the `preview/` and `v0.32` path as well so that this change is carried forward?

Reply:
Sure..